velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -271,6 +271,13 @@ class PlanBuilder {
       connector::CommitStrategy commitStrategy,
       const RowTypePtr outputType);
 
+  /// Add a TableWriteMergeNode.
+  ///
+  /// @param outputType The output type of table write merge node which consists
```

Comment:
Can we remove this parameter? It seems like there is not much freedom in what can be specified here.